.github/workflows/docker-build.yaml

```diff
@@ -125,14 +125,22 @@ jobs:
             (
                 github.event.pull_request.head.repo.full_name == 'shopsys/shopsys' ||
                 github.ref_protected == true
-            )
+            ) &&
+            needs.variables.outputs.DOCKER_PHP_FPM_REPOSITORY_TAG != ''
         name: Build PHP-FPM image
         runs-on: ubuntu-22.04
         needs: variables
         permissions:
             contents: read
             packages: write
         steps:
+            -   name: Cache Docker layers
+                uses: actions/cache@v3
+                with:
+                    path: /tmp/.buildx-cache
+                    key: ${{ runner.os }}-docker-php-fpm-${{ hashFiles('project-base/app/docker/php-fpm/Dockerfile') }}-${{ hashFiles('composer.json') }}
```

Comment:
the cache key doesn't seem right

![Snímek obrazovky 2024-10-30 v 12 12 33](https://github.com/user-attachments/assets/088099ce-dc21-40c2-9e02-a347bf134e88)